### PR TITLE
Document difference between Symfony 2 and 3 form types

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Addresses part 1 of #83.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,18 +110,19 @@ Phone numbers can be deserialized from an international format by setting the ty
 
 ### Using `libphonenumber\PhoneNumber` objects in forms
 
-You can use the `tel` form type to create phone number fields. There are two widgets available.
+You can use the `PhoneNumberType` (`tel` for Symfony 2) form type to create phone number fields. There are two widgets available.
```
Comment (xabbuh) [inline README.md]: The class name should be used in Symfony 2.8 too (you could still use the alias but it will trigger deprecations).

Comment (thewilkybarkid) [inline README.md]: Yeah, I was just trying to not have too many variants in the documentation (and deprecations are shown to the user). Might reword it to `tel for Symfony <= 2.7` though
